src/parsing.ts

```diff
@@ -76,17 +76,33 @@ export interface IParsedNode {
   children: IParsedNode[];
 }
 
-export const parseSource = (text: string) => {
+export const parseSource = (text: string, fsPath?: string) => {
   const ast = parse(text, acornOptions);
 
+  // A list of tests to see if a function call is a "test"
   const idTests: ExtractTest[] = [];
 
   const stack: { node: Node; r: IParsedNode }[] = [];
   stack.push({ node: undefined, r: { children: [] } } as any);
 
   traverse(ast as Node, {
     enter(node) {
-      if (node.type === C.ImportDeclaration && node.source.value === C.NodeTest) {
+      if (node.type === C.ImportDeclaration && node.source.value === "../utils") {
```

Comment:
Instead of this hard-coded value here, it would read the setting and determine if the import matches the configuration.